examples/with-fela/pages/_document.js

```diff
@@ -1,12 +1,20 @@
 import Document, { Head, Main, NextScript } from 'next/document'
 import { renderToSheetList } from 'fela-dom'
-import felaRenderer from '../fela-renderer'
+
+import FelaProvider from '../FelaProvider'
+import getFelaRenderer from '../getFelaRenderer'
 
 export default class MyDocument extends Document {
   static getInitialProps ({ renderPage }) {
-    const page = renderPage()
-    const sheetList = renderToSheetList(felaRenderer)
-    felaRenderer.clear()
+    const serverRenderer = getFelaRenderer()
+
+    const page = renderPage(App => props => (
+      <FelaProvider renderer={serverRenderer}>
+        <App />
```

Comment:
I'm pretty sure we don't pass props, but to make sure it keeps working in the future:

```
<App {...props} />
```